src/app/login/login.demo.html

```diff
@@ -28,8 +28,8 @@ <h5 class="hint">Use your Company ID to sign in or create one now</h5>
                     </label>
                 </div>
                 <div class="error active">
-                    Invalid user name or password
-                </div>
+                                Invalid user name or password
```

Comment:
Formatting change?

Reply:
Might be a rebase change. I will un-indent the lines.